### ISSUE TITLE
ValueError: Unknown status message: 'WARNING'

### ISSUE THREAD

**Issue by rpdelaney** [isislovecruft/python-gnupg#172]
```
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/home/ryan/.local/lib/python3.5/site-packages/gnupg.py", line 835, in _read_response
    result.handle_status(keyword, value)
  File "/home/ryan/.local/lib/python3.5/site-packages/gnupg.py", line 588, in handle_status
    Verify.handle_status(self, key, value)
  File "/home/ryan/.local/lib/python3.5/site-packages/gnupg.py", line 313, in handle_status
    raise ValueError("Unknown status message: %r" % key)
ValueError: Unknown status message: 'WARNING'
```

Happens for me when the gpg-agent is out of date:

```
gpg: WARNING: server 'gpg-agent' is older than us (2.1.15 < 2.1.16)
```

**Comment by isislovecruft** [comment]
I'll gladly take a patch which takes the contents of the `WARNING` status and calls `log.warn()` with it.

**Comment by isislovecruft** [comment]
Fixed in python-gnupg version 2.2.0.
